cmd/minikube/cmd/ssh.go

```diff
@@ -28,7 +28,7 @@ import (
 
 // sshCmd represents the docker-ssh command
 var sshCmd = &cobra.Command{
-	Use:   "ssh",
+	Use:   "ssh ['COMMAND' or -- COMMAND]",
```

Comment:
@hferentschik 

so can we use [ -- COMMAND ]  ? 

Here, main issue is when we try to run command containing `-` inside the vm as
`minishift ssh df -h`. Here, cobra treat `-h` as the root level help and hence print help instead of actual command.

So, here we need something to escape that and looks like cobra allow `-- COMMAND` approach.
Check this https://github.com/spf13/cobra/blob/master/cobra_test.go#L128.

I actually added quote since it was working locally for me but may not work in different shell as you mentioned.

WDYT ?